playground/ssr-deps/forwarded-export/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "forwarded-export",
   "private": true,
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "dependencies": {
+    "object-assigned-exports": "file:../object-assigned-exports"
```

Comment:
This package references `object-assigned-exports` but doesn't specify deps. Added here.